### PR TITLE
Compile head.bin from head.S

### PR DESCRIPTION
**Pull request by jforissier**
Signed-off-by: Jerome Forissier <jerome.forissier@linaro.org>

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
Can't `rpi3/firmware/head.bin` be removed now?

**Comment by jforissier** [comment]
It *is* removed by this patch ;)

**Comment by jenswi-linaro** [comment]
Aha. :-)

File: rpi3.mk [hunk 1]
```diff
@@ -105,6 +105,16 @@ u-boot:
 u-boot-clean:
 	$(U-BOOT_EXPORTS) $(MAKE) -C $(U-BOOT_PATH) clean
 
+head-bin: $(RPI3_HEAD_BIN)
```
Comment (jenswi-linaro) [inline rpi3.mk]: Something should depend on this the `head-bin` target

Comment (jbech-linaro) [inline rpi3.mk]: Shouldn't it be the u-boot target that has a dependency?
I.e.
```Makefile
.PHONY: u-boot                                                                                                                     
u-boot:  $(RPI3_HEAD_BIN)
...
```

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by igoropaniuk** [comment]
I was wondering, what was a profit of having separated target for head.bin, and why not to compile it inside rpi3-firmware target, but then noticed that we have two separate firmware paths (a bit confusing) `RPI3_FIRMWARE_PATH`/`RPI3_STOCK_FW_PATH`, and, actually, this `head.S` doesn't relate to this rpi3-firmware target.

`Reviewed-by: Igor Opaniuk <igor.opaniuk@linaro.org>`

**Pull request merged**